source/_integrations/ozw.markdown

```diff
@@ -65,3 +65,24 @@ this operation.
 | Service Data Attribute | Required | Description                                        |
 | ---------------------- | -------- | -------------------------------------------------- |
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
+Valid usercodes are at least 4 digits.
+
+| Service Data Attribute | Required | Description                             |
+| ---------------------- | -------- | --------------------------------------- |
+| `entity_id`            | yes      | Lock entity to set the usercode.        |
+| `code_slot`            | yes      | The code slot to set the usercode into. |
+| `usercode`             | yes      | The code to set in the slot.            |
+
+### Service `ozw.clear_usercode`
+
+This service will clear the usercode in code_slot X.
+Valid code_slots are 1-254.
+
+| Service Data Attribute | Required | Description                               |
+| ---------------------- | -------- | ----------------------------------------- |
+| `entity_id`            | yes      | Lock entity to clear the usercode.        |
```

Comment:
This service uses `async_register_entity_service` to register, thus the `entity_id` is optional as an area can be targeted instead.